src/test/java/codeu/controller/ChatServletTest.java

```diff
@@ -206,8 +206,11 @@ public void testDoPost_CleansHtmlContent() throws IOException, ServletException
 
     ArgumentCaptor<Message> messageArgumentCaptor = ArgumentCaptor.forClass(Message.class);
     Mockito.verify(mockMessageStore).addMessage(messageArgumentCaptor.capture());
+    
+    // so this is really weird. I have to include a newline escape key before the <b> tag for this test to pass
```

Comment:
I probably wouldn't leave this comment in. It's probably enough to leave the question in your pull request description, or to send out a chat message. This is weird though, I'll play around a bit and get back to you.

Reply:
Looks like this is a JSoup quirk: https://stackoverflow.com/questions/12503117/jsoup-line-feed

(Googling "jsoup.clean adding line breaks" returns a bunch of other results if you're curious)

It's up to you whether you want to keep the newline in the test or use the advice in the above link to get rid of it. I think the final HTML will look okay either way.

Reply:
Thanks so much for the link. I felt like I was banging my head against a wall trying to find the issue on stackoverflow. I fixed the issue using the stackoverflow solution just because I feel like it'll be better for writing sensible test-cases in the long run.